src/command/graph/check.rs

```diff
@@ -29,6 +32,21 @@ pub struct Check {
     #[structopt(long, short = "s", parse(try_from_str = parse_schema_source))]
     #[serde(skip_serializing)]
     schema: SchemaSource,
+
+    /// Minimum number of requests within the time window for a query to be
+    /// considered
```

Comment:
I'm a bit confused by this wording - what does "to be considered" mean in this context?

Reply:
```
/// Minimum number of times an operation must have happened within the time 
/// window for checks be able to fail on that operation
```

Does this make more sense?

Reply:
```
/// The minimum number of times a query or mutation must have been executed
/// in order to be considered in the check operation
```

Reply:
Makes sense! I updated the percentage flag to use similar wording